styles/index.css

```diff
@@ -96,12 +96,15 @@ nav a:hover {
 }
 
 footer {
+  display: flex;
+  align-items: center;
   text-align: center;
-  padding: 3px;
+  justify-content: center;
+  padding: 10px;
   background-color: #cedebd;
   color: white;
   position: relative;
-  font-size: 22px;
+  font-size: 40px;
   bottom: 0;
   left: 0;
   transform: translate(calc(50vw - 50%), -50%);
```

Comment:
please add `margin-top: 50px; `in `footer ` -to give space between above section.